components/services/brave_content_browser_overlay_manifest.cc

```diff
@@ -3,6 +3,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#if !defined(OS_ANDROID)
```

Comment:
not sure if related to rewards

Reply:
Some code in this PR is not related to Rewards and Ads, but necessary to make it buildable on Android.

Reply:
We don't need this change, reverted it.